### PR TITLE
Add new test for empty observables

### PR DESCRIPTION
**Pull request by YevhenLysen**

### CODE REVIEW

**Review by GeorgiyII — APPROVED** [review]

File: tests/functional/tests/test_empty_observables.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      'sha256'),
+     ('d41d8cd98f00b204e9800998ecf8427e', 'md5'),
+     ('WininetStartMutex', 'mutex'),
+     (r'C:\Users\Users\Downloads\Malware', 'file_path'),
+     ('test.exe', 'file_name'))
+)
+def test_positive_smoke_empty_observables(
+        module_headers, observable, observable_type):
+    """ Perform testing for enrich observe observables endpoint to check that
+     observable, on which Qualys  doesn't have information, will
```
Comment (mistergreen22) [inline tests/functional/tests/test_empty_observables.py:20]: hey, i think you have some extra space on this line

File: tests/functional/tests/test_empty_observables.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    observables = [{"value": observable, "type": observable_type}]
+    response_from_all_modules = enrich_observe_observables(
+        payload=observables,
+        **{'headers': module_headers})
+
+    qualys_data = response_from_all_modules['data']
+
+    response_from_qualys_module = get_observables(
+        qualys_data, 'Qualys IOC')
+    assert response_from_qualys_module['module'] == 'Qualys IOC'
```
Comment (mistergreen22) [inline tests/functional/tests/test_empty_observables.py:43]: Use please `MODULE_NAME` variable for assertion

File: tests/functional/tests/test_empty_observables.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    Expectedresults:
+        1. Response body contains empty data dict from Qualys module
+
+    Importance: Critical
+    """
+
+    observables = [{"value": observable, "type": observable_type}]
+    response_from_all_modules = enrich_observe_observables(
+        payload=observables,
+        **{'headers': module_headers})
```
Comment (mistergreen22) [inline tests/functional/tests/test_empty_observables.py:37]: move `)` to new line pls

File: tests/functional/tests/test_empty_observables.py (hunk 1 above)
Comment (mistergreen22) [reply tests/functional/tests/test_empty_observables.py:20]: and redundant space between `Qualys   doesn't` i think

File: tests/functional/tests/test_empty_observables.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+    qualys_data = response_from_all_modules['data']
+
+    response_from_qualys_module = get_observables(
+        qualys_data, 'Qualys IOC')
```
[the remark this answers is not in the document]
Comment (mistergreen22) [reply tests/functional/tests/test_empty_observables.py:42]: And you can fit it on one line

File: tests/functional/tests/test_empty_observables.py (hunk 1 above)
Comment (mistergreen22) [reply tests/functional/tests/test_empty_observables.py:20]: and remove space between docstring symbol and starting your explanation `"""Perform`

File: tests/functional/tests/constants.py [hunk 5]
```diff
@@ -0,0 +1,2 @@
+MODULE_NAME = 'Qualys IOC'
```
Comment (mistergreen22) [inline tests/functional/tests/constants.py:1]: Hey) You don't need to create new "constants" file because it's already exist! please take a look in source code
image
You need just import this constant. Probably you have the old version of the repo or something like this, try to update develop branch and merge it in to yours for resolving conflict.

File: tests/functional/library/constants.py [hunk 6]
```diff
@@ -8,3 +8,4 @@
     'C:\\Windows\\SYSTEM32\\ntdll.dll': 'file_path',
     'photoview1sp.jpg': 'file_name',
 }
+MODULE_NAME = 'Qualys IOC'
```
Comment (mistergreen22) [inline tests/functional/library/constants.py:11]: Wrong constants file, you use tests/functional/library/constants.py  but you need tests/functional/tests/constants.py, in this file we already have constants that you want, didn't need to create new
